include/courtier/GWebsocketConsumerT.hpp

```diff
@@ -1508,6 +1513,445 @@ class GWebsocketConsumerT
 	 //-------------------------------------------------------------------------
 };
 
+
+// Add just another consumer
+
+template<typename processable_type>
+class GWebsocketConsumerPT
+	: public Gem::Courtier::GBaseConsumerT<processable_type> // note: GBaseConsumerT<> is non-copyable
+	  , public std::enable_shared_from_this<GWebsocketConsumerPT<processable_type>>
+{
+	 //-------------------------------------------------------------------------
+	 // Simplify usage of namespaces
+	 using error_code = boost::system::error_code;
+
+public:
+	 //-------------------------------------------------------------------------
+	 /** @brief The default constructor */
+	 GWebsocketConsumerPT() = default;
+  
+         GWebsocketConsumerPT(int io_context_pool_size)
```

Comment:
Formatting is funny, in my editor. Probably spaces vs. tabs. We need a suitable clang format file